pdr_backend/analytics/predictoor_dashboard/dash_components/view_elements.py

```diff
@@ -129,7 +129,6 @@ def get_metric(label, value, value_id):
 def get_layout():
     return html.Div(
         [
-            dcc.Store(id="data-folder"),
             dcc.Store(id="feeds-data"),
             dcc.Store(id="predictoors-data"),
```

Comment:
I think this 2 should also be removed now that the data is stored inside the app